core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java

```diff
@@ -804,56 +797,7 @@ public Category getCategory() {
         }
     }
 
-    /**
-     * {@link PasswordEncoder} based on SHA-256 and random salt generation.
-     *
-     * <p>
-     * The salt is prepended to the hashed password and returned. So the encoded password is of the form
-     * {@code SALT ':' hash(PASSWORD,SALT)}.
```

Comment:
Did not see an easy way to translate this to Spring Security. There _is_ a `StandardPasswordEncoder` and a `MessageDigestPasswordEncoder` but these are deprecated and considered insecure. Since this password format was last stored eight years ago, it does not seem worth saving.